src/commands/print-stacks.ts

```diff
@@ -52,13 +52,16 @@ export class PrintStacksCommand extends BaseCliCommand<IPrintStacksCommandArgs>
         const cfnBinder = new CloudFormationBinder(command.stackName, template, state, parameters, undefined, undefined, undefined, undefined, undefined, undefined, false, undefined, command.resolver);
 
         const bindings = await cfnBinder.enumBindings();
+        console.log(bindings);
```

Comment:
were these intentional? or leftovers from debugging?

Reply:
unintentional. This is still a draft I am currently working on. :)